app/views/usage.scala.html

```diff
@@ -203,6 +204,37 @@ <h2>Current Version(s)</h2>
                     </p>
                 </article>
                 <br/>
+                <p>
+                    <a name='completion'></a>
+                    <h2>Shell Completion</h2>
+                    <p>
+                    With SDKMAN!'s rich set of commands and subcommands, it can sometimes be hard to remember which
+                    commands are supported or which subcommands are allowed by a command.
+                    Wouldn't it be nice if the shell would autocomplete them for you?
+                    Luckily, SDKMAN! comes packaged with shell completions for both Bash and Zsh (not supported yet).
+                    To enable shell completion for the current session, just issue the following command:
+                    <pre><code>source <(sdk env bash)</code></pre>
+                    Typing <code>sdk</code> followed by two TABs will now bring up the list of possible top-level commands:
+<pre><code>
+$ sdk &lt;tab&gt;&lt;tab&gt;
+broadcast   current     env         help        install     offline     uninstall   upgrade     version
+completion  default     flush       home        list        selfupdate  update      use
+</code></pre>
+                    When double-tapping after entering a top-level command, the list of subcommands is shown:
+<pre><code>
+$ sdk env &lt;tab&gt;&lt;tab&gt;
+clear    init     install
+</code></pre>
+                    Note that shell completion is currently not supported for candidates and versions.
+                    <p>
+                    </p>
+                    To permanently enable shell completion, add it to your <code>.bashrc</code> (Linux)
+                    <code>.bash_profile</code> (Mac), or <code>.profile</code> (Unix), right after the
```

Comment:
On Unix based systems we [use the `.bashrc`](https://github.com/sdkman/sdkman-hooks/blob/1079a92c33e0cd663111b9de03cfcf50b72fc37d/app/views/install.scala.txt#L286) to initialise the bash profile.